scripts/generate-report.py

```
@@ -233,9 +233,17 @@ def fetch_from_vps():
     )
-    print("Connecting to VPS and pulling data...")
+    vps_host = os.environ.get("REPORT_VPS_HOST")
+    if not vps_host:
+        print(
+            "Set REPORT_VPS_HOST to your VPS ssh host/alias to pull live data "
+            "(no host is baked in); or use the local-CSV path instead.",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+    print(f"Connecting to {vps_host} and pulling data...")
     result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)